simplecpp.cpp

```diff
@@ -788,6 +788,35 @@ void simplecpp::TokenList::readfile(Stream &stream, const std::string &filename,
             while (stream.good() && ch != '\r' && ch != '\n') {
```

Comment:
I am thinking if something like this could work:
```
    char lastChar = ' ';
    while (stream.good() && (lastChar == '\\' || (ch != '\r' && ch != '\n')) {
        currentToken += ch;
        lastChar = ch;
        ch = stream.readChar();
    }
```